expr/func_constantline.go

```diff
@@ -0,0 +1,64 @@
+package expr
+
+import (
+	"fmt"
+
+	"github.com/grafana/metrictank/api/models"
+	"github.com/grafana/metrictank/schema"
+)
+
+type FuncConstantLine struct {
+	value float64
+	from  uint32
+	to    uint32
+}
+
+func NewConstantLine() GraphiteFunc {
+	return &FuncConstantLine{}
+}
+
+func (s *FuncConstantLine) Signature() ([]Arg, []Arg) {
+	return []Arg{
+			ArgFloat{key: "value", val: &s.value},
+		}, []Arg{
+			ArgSeriesList{},
+		}
+}
+
+func (s *FuncConstantLine) Context(context Context) Context {
+	s.from = context.from - 1
+	s.to = context.to - 1
+	return context
```

Comment:
within MT, `from` is always inclusive, so the first point can have ts from, but nothing earlier, as that would be out of range.
also the names from/to are by convention used to mark the time window.
if you want to track the first and last timestamp to include, better names would be first and last

Reply:
Code was hitting https://github.com/grafana/metrictank/blob/master/api/graphite.go#L1363-L1367, and I was trying to line up with graphite's constantLine behavior, which was inclusive on both ends (even though normally, graphite is from exclusive, to inclusive).

Reply:
I see. in that case i propose we just add a comment explaining that graphite does it non-standard and we do the same.